src/config.json

```diff
@@ -43,28 +45,32 @@
                     "image": "assets/images/locations/dominican-republic.svg",
                     "mapUrl": "http://coastalresilience.org/project/dominican-republic/",
-                    "mapCenter": [18.73, -70.16]
+                    "mapCenter": [18.73, -70.16],
+                    "mapZoom": 6
                 },
                 {
                     "title": "Grenada, St. Vincent, and the Grenadines",
                     "description": "Demonstrating that governments and communities of small island states can enhance their resilience to climate change by protecting, restoring and effectively managing their marine and coastal ecosystems and strengthening local capacity for adaptation",
                     "image": "assets/images/locations/grenada.svg",
                     "mapUrl": "http://maps.coastalresilience.org/gsvg/",
-                    "mapCenter": [12.83, -61.04]
+                    "mapCenter": [12.83, -61.04],
+                    "mapZoom": 6
                 },
                 {
                     "title": "Jamaica",
                     "description": "The Small Island Developing States (SIDS) of the Caribbean are among the world’s most vulnerable to the impacts of climate change. While these losses are expected to grow, ecosystem-based adaptation can significantly reduce risk and avert economic losses. This project, a collaboration between TNC and IFRC, launches in January, 2018.",
                     "image": "assets/images/locations/jamaica.svg",
                     "mapUrl": "http://coastalresilience.org/project/jamaica/",
-                    "mapCenter": [18.10, -77.29]
+                    "mapCenter": [18.10, -77.29],
+                    "mapZoom": 6
                 },
                 {
                     "title": "U.S Virgin Islands",
                     "description": "In the US Virgin Islands, high resolution satellite imagery and GIS-based models are being used to identify and prioritize high risk and vulnerable coastal and marine sites subject to the effects of sea level rise, increasing storm surge and intensity, and altered precipitation patterns. The impacts of future SLR and storm surge scenarios are being validated by incorporating community perception, knowledge and historic events as a baseline to better understand how ecosystem-based adaptation solutions can help increase people and nature’s resilience to these impacts.",
                     "image": "assets/images/locations/virgin-islands.svg",
                     "mapUrl": "http://maps.coastalresilience.org/usvi/",
-                    "mapCenter": [17.72, -64.78]
+                    "mapCenter": [17.72, -64.78],
+                    "mapZoom": 6
```

Comment:
Once these changes are in we should update the configuration docs in the wiki.

Reply:
Good call.